### ISSUE TITLE
Cache component versions locally instead of making an HTTP request

### ISSUE THREAD

**Issue by fstanis** [ampproject/docs#2410]
It seems that we're getting component versions from a remote endpoint:

https://github.com/ampproject/docs/blob/5fe8a1c60d3e1230022125b43b4e774be05c6ee4/pages/extensions/amp_example_preview/util/amp_component_versions.py#L3-L5

Since `https://playground.amp.dev/api/amp-component-versions` doesn't seem to work since ampbyexample.com doesn't work, this prevents `gulp develop` from working now.

Since component versions don't change often enough to justify making a request like this, we should consider caching component versions locally and statically generating them. Or at the very least hosting them outside of `ampbyexample.com`.

**Comment by fstanis** [comment]
Also in:

https://github.com/ampproject/docs/blob/5fe8a1c60d3e1230022125b43b4e774be05c6ee4/pages/extensions/amp_dependency_injector/amp_dependency_injector.py#L10-L12

**Comment by fstanis** [comment]
For those trying to test the website locally: one quick workaround is to replace both occurances of

```python
COMPONENT_VERSIONS = requests.get(COMPONENT_VERSIONS_URL).json()
```

with

```python
COMPONENT_VERSIONS = {}
```

**Comment by matthiasrohmer** [comment]
Thanks for the report, @fstanis and @morsssss! That amp.dev is relying on requests to remote servers for builds has been bothering me for a while already. Also because it makes offline development quite difficult. So updating this is a good thing.

The actual issue here though seems to be that ampbyexample.com's SSL certificate has expired. At least that is what `curl` says. Does anyone else besides Sebastian has access to ABE's infrastructure and can fix this?

**Comment by fstanis** [comment]
@ithinkihaveacat fixed the certificate just now.

I'm working on a PR to remove the request from amp.dev.

**Comment by fstanis** [comment]
#2414 should fix this and also, ampproject/amphtml#23150 removes almost all remaining links to ampbyexample.com.
